lstchain/reco/volume_reducer.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+"""
+**TEMPORARY** module with functions to perform a volume reduction to LST data:
+
+This is a temporary implementation of the 'tailcut and dilation' volume reduction method.
+The volume reduction produces reduced waveforms and write them in the DL0 container.
+
+To date, the ** DL0 container is overwritten ** if the volume reduction is applied.
+
+
+Usage:
+In the configuration file:
+(...)
+
+"volume_reducer":{
+    "algorithm": "zero_suppression_tailcut_dilation",
+    "parameters": { # add here the desired tailcut parameters. See function's help for default parameters.
+    }
+ }
+
+(...)
+"""
+from ctapipe.image.cleaning import tailcuts_clean, dilate
+
+__all__ = ['get_volume_reduction_method',
+           'apply_volume_reduction',
+           'zero_suppression_tailcut_dilation'
+           ]
+
+
+def get_volume_reduction_method(config_file):
+    """
+    Checks in the configuration file if a volume reduction method has been set.
+
+    Parameters
+    ----------
+    config_file: dict
+        The configuration file used in the parent code.
+
+    Returns
+    -------
+    algorithm: str
+        Volume reduction algorithm name
+    """
+    if config_file['volume_reducer']['algorithm'] is None:
+        algorithm = None
+    else:
+        algorithm = config_file['volume_reducer']['algorithm']
+
+    return algorithm
+
+
+def apply_volume_reduction(event, vol_reduction_algorithm, config):
```

Comment:
If you need to pass the config, then passing the `vol_reduction_algorithm` arguments is redundant (since it's given by the config), no?

Reply:
Even more so if you pass only a string and not the function itself.